DuckDuckGo/FireAnimation.swift

```diff
@@ -30,22 +35,53 @@ struct FireAnimation {
         }
         
         let animationContainer = UIView(frame: window.frame)
```

Comment:
It looks great, but if you rotate while it's in flight it doesn't adjust size.

If you add ` animationContainer.autoresizingMask = .flexibleWidth` here and the same for the fiewView it will then resize automatically on rotation.